packages/cli/src/commands/deploy/args.ts

```diff
@@ -1,13 +1,14 @@
 import chalk from 'chalk';
 import logo from '../../util/output/logo';
-import { getPkgName } from '../../util/pkg-name.ts';
+import { getPkgName } from '../../util/pkg-name';
 
 export const help = () => `
   ${chalk.bold(`${logo} ${getPkgName()}`)} [options] <command | path>
 
   ${chalk.dim('Commands:')}
 
     ${chalk.dim('Basic')}
+    
```

Comment:
Was this newline intentional?

Reply:
Create a speration between the words `Commands` and `Bacis` 👍

Reply:
I removed it.